src/ripple/app/misc/impl/LoadFeeTrack.cpp

```diff
@@ -120,17 +104,49 @@ void lowestTerms(T1& a,  T2& b)
         x = y;
         y = t;
     }
-    a /= x;
-    b /= x;
+    return x;
 }
 
 // Scale using load as well as base rate
-std::uint64_t
-scaleFeeLoad(std::uint64_t fee, LoadFeeTrack const& feeTrack,
+Drops64
+scaleFeeLoad(FeeUnit64 fee, LoadFeeTrack const& feeTrack,
     Fees const& fees, bool bUnlimited)
 {
-    if (fee == 0)
-        return fee;
+    auto lowestTerms1 = [](auto& a, auto& b)
```

Comment:
nit: would prefer a better name than `lowestTerms1` and `lowestTerms2`, or find a way to combine the two functions with `if constexpr` something like... 
``` 
auto lowestTerms = [](auto& a, auto& b) {
    auto get_value = [](auto v) {
        if constexpr (std::is_same_v<decltype(v), TYPE_WITH_VALUE_MEMBER_FUNC>) {
            return v.value();
        } else {
            return v;
        }
    };

    if (auto const g = gcd(get_value(a), get_value(b))) 
    { 
        a /= g;
        b /= g;
    }
};
```
but no big deal...